requirements.txt

```diff
@@ -1,2 +1,3 @@
 docker-py
 dockerfile-parse
+#pygit2
```

Comment:
Great find! Unfortunately it fails on Python 2.6: https://travis-ci.org/projectatomic/atomic-reactor/jobs/75398810